app/tasks/doctor.php

```diff
@@ -134,7 +134,9 @@
         }
 
         try {
-            $mail = $register->get('smtp'); /* @var $mail \PHPMailer\PHPMailer\PHPMailer */
+            /** @var PHPMailer\PHPMailer\PHPMailer $mail */
+            $mail = $register->get('smtp'); 
+            $mail->SMTPDebug = PHPMailer\PHPMailer\SMTP::DEBUG_SERVER;
```

Comment:
@eldadfux, [you said it would be good to enable debugging in doctor](https://github.com/appwrite/appwrite/issues/1390#issuecomment-880077189), but will it be too messy?

Reply:
Yeh, this feels quite messy. We can try and have a different command for doing this.

Something like:
`docker exec appwrite debug smtp`